### PR TITLE
/azure

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Added /azure section

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/azure
- Ensure that the modal contact form works
-  http://0.0.0.0:8001/azure/pro
-  http://0.0.0.0:8001/azure/contact-us
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes #2177
Fixes #2178

### CODE REVIEW

**Comment by SirSamTumless** [comment]
[Ubuntu_Pro_Azure_03.04.20.pdf.zip](https://github.com/canonical-web-and-design/ubuntu.com/files/4428838/Ubuntu_Pro_Azure_03.04.20.pdf.zip)

PDF

File: templates/shared/forms/interactive/_azure.html [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+          </div>
+          <div class="col-4 u-sv3">
+            <input type="checkbox" id="project-development-and-CI-CD">
+            <label for="project-development-and-CI-CD">Development and CI/CD</label>
+            <input type="checkbox" id="project-standard-production">
+            <label for="project-standard-production">Standard Production</label>
+            <input type="checkbox" id="project-mission-critical-production">
+            <label for="project-mission-critical-production">Mission-critical Production</label>
+            <input type="checkbox" id="project-mission-regulated-workloads">
+            <label for="project-mission-regulated-workloads">Mission-critical Production</label>
```
Comment (sowasred2012) [inline templates/shared/forms/interactive/_azure.html:41]:
```suggestion
            <label for="project-mission-regulated-workloads">Regulated workloads</label>
```

File: templates/azure/index.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+      </div>
+    </div>
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">Ubuntu Pro for Azure</h3>
+      <p class="p-heading--two">Starting at 0.1c/hour</p>
+      <hr class="u-sv1">
+      <div class="p-card__content">
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">Includes all Ubuntu optimisations and updates of the free version</li>
+          <li class="p-list__item is-ticked">Expanded security covering application components delivered with Ubuntu</li>
```
Comment (sowasred2012) [inline templates/azure/index.html:100]: The copy doc has this item as "Expanded security covering 26,000 additional application components delivered with Ubuntu"

File: templates/azure/pro.html [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <h2>Broadest open source security coverage keeps CISOs sleeping at night</h2>
+      <p>Ubuntu Pro includes security coverage for the entire collection of software packages shipped with Ubuntu, tracking high and critical CVEs. This makes it ideal for companies that have embraced open source for their new products and need to ensure security for production and mission-critical workloads.</p>
+      <p>One key reason why developers prefer Ubuntu is that it includes a huge set &mdash; in total, over 26,000 packages &mdash; of upstream open source known as ‘universe’. Universe includes workloads like NGINX, MongoDB, Redis, Kafka, but also language ecosystems like Python, Java, Ruby and Javascript/Node.js. With Ubuntu Pro's built-in CVE tracking and patching, CIOs and security officers can be confident they can roll these packages out to production.</p>
+    </div>
+  </div>
+  <div class="col-4">
+    {% include "azure/shared/_azure-pie-chart.html" %}
```
Comment (sowasred2012) [inline templates/azure/pro.html:146]: This is causing a TemplateNotFound error, I can't see this page at all

File: templates/azure/pro.html [hunk 4]
```diff
@@ -0,0 +1,299 @@
...
+      <p><a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic?tab=Overview" class="p-button--positive"><span class="p-link--external">Launch now</span></a></p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 16.04 LTS</h3>
+      <p>The most commonly deployed Ubuntu LTS on the cloud today, Ubuntu 16.04 LTS released in 2016 with the 4.4 kernel, and is covered with Ubuntu Pro through 2024.</p>
+      <p><a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-xenial?tab=Overview" class="p-button--positive"><span class="p-link--external">Launch now</span></a></p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 14.04 LTS</h3>
+      <p>Ubuntu Pro 14.04 LTS Ubuntu 14.04 LTS shipped in 2014 with the 3.13 kernel. Ubuntu Pro extends our security coverage to 2022, giving you plenty of time to prepare and upgrade.</p>
```
Comment (sowasred2012) [inline templates/azure/pro.html:131]:
```suggestion
      <p>Ubuntu 14.04 LTS shipped in 2014 with the 3.13 kernel. Ubuntu Pro extends our security coverage to 2022, giving you plenty of time to prepare and upgrade.</p>
```

**Pull request merged**